docs/scenarios_authoring/Spel.md

```diff
@@ -240,6 +240,10 @@ If you need to invoke the same method in many places, probably the best solution
 
 ### Type conversions
 
+It is possible to convert from a type to another type and this can be done by implicit and explicit conversion.
+
+#### Implicit conversion
+
 SpEL has many built-in implicit conversions that are available also in Nussknacker. Mostly conversions between various
```

Comment:
I would explain what "implicit conversion" means. For the non-programmer this may be not fully obvious. Something like: 

_Implicit means when finding the "input value" of type "input type" (see the table below) in the context where "target type" is expected, Nussknacker will try to convert the type of the "input value" to the "target type"._ 

(After the second sentence)

Reply:
Done

Reply:
I am sorry, I missed out "that". 

_implicit conversion means **that** _